administrator/language/en-GB/plg_system_actionlogs.ini

```
@@ -12,8 +12,8 @@ PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Action Log - Joomla"
 PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="The %s plugin is disabled."
 PLG_SYSTEM_ACTIONLOGS_LOG_DELETE_PERIOD="Days to delete logs after"
 PLG_SYSTEM_ACTIONLOGS_LOG_DELETE_PERIOD_DESC="Enter 0 if you don't want to delete the logs."
-PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS="Send notifications for User Actions Log"
-PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS_DESC="Send a notifications of users' actions log to your email"
+PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS="Send notifications"
+PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS_DESC="Send notifications of a users' actions to your email."
```

```suggestion
PLG_SYSTEM_ACTIONLOGS_NOTIFICATIONS_DESC="Send notifications of users' actions to your email."
```
`users'` implies more than one person.

---

it is isn't it?

It will send the an email notification of the actions (plural) of all users (plural)

---

Ahh I see what @Quy is saying. The "a" needs removed since "users'" is plural.

---

doh